recipes/face/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "face" %}
+{% set version = "19.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "38c94ec17a4d6f9628f094b697faca0f802f4028071ce8cbdb3116d4cde772a3"
+
+build:
+  number: 0
```

This looks like it can be noarch?
```suggestion
  number: 0
  noarch: python
```